dowhy/causal_refuters/evalue_sensitivity_analyzer.py

```diff
@@ -0,0 +1,301 @@
+import logging
+import numpy as np
+import pandas as pd
+
+from dowhy.causal_estimator import CausalEstimate
+from dowhy.causal_estimators.linear_regression_estimator import (
+    LinearRegressionEstimator,
+)
+from dowhy.causal_estimators.causalml import Causalml
+from dowhy.causal_estimators.econml import Econml
+from dowhy.causal_estimators.instrumental_variable_estimator import (
+    InstrumentalVariableEstimator,
+)
+from dowhy.causal_estimators.regression_discontinuity_estimator import (
+    RegressionDiscontinuityEstimator,
+)
+from dowhy.causal_estimators.two_stage_regression_estimator import (
+    TwoStageRegressionEstimator,
+)
+
+
+class EValueSensitivityAnalyzer:
+    """
+    This class computes Ding & VanderWeele's E-value for unmeasured confounding. The E-value is the minimum
+    strength of association on the risk ratio scale that an unmeasured confounder would need to have with
+    both the treatment and the outcome, conditional on the measured covariates, to fully explain away a specific
+    treatment-outcome association.
+
+    See: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4820664/ and
+    https://dash.harvard.edu/bitstream/handle/1/36874927/EValue_FinalSubmission.pdf.
+    The implementation is based on the R package: https://github.com/cran/EValue.
+
+    :param estimate: CausalEstimate
+    :param outcome_var: Outcome variable values
+    :param true: A number to which to shift the observed estimate to. Defaults to 1 for ratio measures (RR, OR, HR)
+    and 0 for additive measures (OLS, MD). (Default = 0)
+    :param delta: The contrast of interest in the treatment/exposure, for estimator of class LinearRegressionEstimator.
```

Comment:
I did not understand what delta means here.  Is it the value of the treatment?

Reply:
`delta` is used to compute the E-value from a linear regression coefficient. From the R package: "the choice of `delta`defines essentially a dichotomization in the exposure between hypothetical groups of subjects with exposures equal to an arbitrary value `c` versus another hypothetical group with exposures equal to `c+delta`"

I think I can remove this parameter and instead simply use `estimate.treatment_value - estimate.control_value`.

Reply:
Sounds good, thanks!